code/modules/projectiles/projectile/bullets.dm

```diff
@@ -53,6 +53,18 @@
 /obj/item/projectile/bullet/heavybullet
 	damage = 35
 
+/obj/item/projectile/bullet/sabot
+	damage = 150
+	var/breakthings = FALSE
+	armour_penetration = -5 //NOT A TYPO
+
+/obj/item/projectile/bullet/sabot/Range()
+	..()
+	damage += 70

```

Comment:
Additional shells will be quite expensive.

40mm buckshot is not actually lethal. It will knock you down but I've discovered it absolutely cannot kill you in a single shot.

However 24 pellets will easily clear anyone out.

Regarding the sabot. Originally it was 500 damage with a massive minus penetration to offset the damage. Unfortunately this is extremely unstable and anyone with armour would be immune completely.

I'm tempted to scrap this, recolour the Sprite to resemble Emp grenades and turn this into a massive EMP grenade.